v2/azure-arm.yaml

```diff
@@ -683,6 +683,18 @@ status:
 #     sub-resource.
 #
 objectModelConfiguration:
+  apimanagement:
```

Comment:
I think to achieve the suggestions I gave in my comments you want the following:
```
  apimanagement:
    2022-08-01:
      AdditionalLocation:
        PublicIpAddressId:
          $armReference: true
      Service:
        $export: true
        $supportedFrom: v2.4.0
      CertificateInformation:
        Expiry:
          $importConfigMapMode: optional
        Subject:
          $importConfigMapMode: optional
        Thumbprint:
          $importConfigMapMode: optional
      HostnameConfiguration:
        CertificatePassword:
          $isSecret: true
        IdentityClientId:
          $importConfigMapMode: optional
      VirtualNetworkConfiguration:
        SubnetResourceId:
          $armReference: true
```

Reply:
Also up above in the `typeTransformers` section, you'll want:
```
  - name: ApiManagementServiceProperties
    group: apimanagement
    property: PrivateEndpointConnections
    remove: true
    because: This property should be marked readonly
```

I think. cc @super-harsh on this if I am incorrect about how we're doing private endpoint connection approvals (I think right now maybe we are not?)

Reply:
Done